GlotPress/gp-translation-helpers
# Add mentions to comment box

#### Problem

On translate.wordpress.org typing @[username] in the comment box in the discussions page does not show a drop down of suggested users but when the correct user name is entered after `@` and the form submitted, the mentioned user gets a notification.

#### Solution

Adapt the solution used on make.wordpress.org and make it work on translate.wordpress.org.

#### Testing instructions

Go to a discussions page for a string(see [example](https://translate.wordpress.org/projects/wp-plugins/woocommerce-product-addon/dev/14400874/es/default/)) and in the comment box start by typing `@`, a list of suggested users should be displayed.

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/2834132/190505521-864fd204-3ac1-4f1f-ac64-bc1644ebf4bb.png">
